repository/cassandra_chat_schema.cql

```diff
@@ -0,0 +1,13 @@
+CREATE KEYSPACE IF NOT EXISTS chat WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1};
+
+DROP TABLE IF EXISTS chat.messages;
+
+CREATE TABLE IF NOT EXISTS chat.messages (
+    message_id      UUID,
+    from_student_id text,
+    to_student_id   text,
+    message_body    text,
+    team_id         text,
+    sent_timestamp  timestamp,
+    PRIMARY KEY ( (from_student_id, to_student_id), sent_timestamp )
+) WITH CLUSTERING ORDER BY (sent_timestamp ASC);
```

Comment:
1. What is the reason for having message_id as UUID if we don't use it. It's not a part of the primary key.
2. We could generalize to be a message for room, group or individual. For that purpose, we can remove to_student_id, and replace it with a room_id.
3. team_id can be replaced with the room_id.
4. room_id can be set as the partition key and sent_timestamp as the clustering key.
5. I would order by sent_timestamp DESC instead of ASC. This would allow you to get n newest messages instead of the first n in a chat.

Reply:
good point, the message_id can be removed. I like the room_id idea!

Reply:
I made an attempt to address this, since this is time sensitive. Can you review the PR please.